File: simu13/src/Teoria.java
public class Teoria extends Corso {
    int codice_aula;
    int numero_ore_settimanali;
    double numer_ore_per_lezione;

    public Teoria(int codice, String tipo, String nomeCorso, String nome_cognome_docente, int codice_aula,
            int numero_ore_settimanali, double numer_ore_per_lezione) {
        super(codice, tipo, nomeCorso, nome_cognome_docente);
        this.codice_aula = codice_aula;
        this.numero_ore_settimanali = numero_ore_settimanali;
        this.numer_ore_per_lezione = numer_ore_per_lezione;
    }

    @Override
    public String toString() {
        return nomeCorso + "\t" + codice + "\t" + nome_cognome_docente + "\t" + tipo + "\t" + codice_aula + "\t"
                + numero_ore_settimanali + "\t" + numer_ore_per_lezione + "\t-" + "\t-" + "\t-";
    }

}

File: 22/Teoria.py
from Corsi import Corsi
class Teoria(Corsi):
    def __init__(self, codice, tipo, nomeDelCorso, nomeCognome,codAula,numeroOreSettimanali,numeroOrePerLezione) -> None:
        super().__init__(codice, tipo, nomeDelCorso, nomeCognome)
        self.codAula = codAula
        self.numeroOreSettimanali = numeroOreSettimanali
        self.numeroOrePerLezione  = numeroOrePerLezione
    
    def __str__(self) -> str:
        return self.nomeDelCorso+"\t"+str(self.codice)+"\t"+self.nomeCognome+"\t"+self.tipo+"\t"+str(self.codAula)+"\t"+str(self.numeroOreSettimanali)+"\t"+str(self.numeroOrePerLezione)+"\t"+"-"+"\t"+"-"+"\t"+"-"